### ISSUE TITLE
VictoryLine is not working when adding animate prop

### ISSUE THREAD

**Issue by boygirl** [FormidableLabs/victory-native#299]
_From @JensonMiralles on April 16, 2018 3:26_

VictoryLine is not working when adding `animate` prop (React Native)

_Copied from original issue: FormidableLabs/victory#972_

**Comment by boygirl** [comment]
@JensonMiralles

This issue is caused by a change in `react-native-svg@^6.2.x`. Since this change `ClipPath` components are not updating their widths which causes animating line and area charts to always be rendered with a 0 width `ClipPath`. 

You can get around this by:

1) Using an earlier version of `react-native-svg`. Compatible versions are described [here](https://github.com/FormidableLabs/victory-native#peer-dependencies-and-version-requirements)

2) Replacing the `groupComponent` in `VictoryLine` with a standard `<G/>` tag. You will not see curtaining transitions (no more 'onLoad' transitions, or nice transitions for changing numbers of data points), but regular animations will work correctly.

**Comment by JensonMiralles** [comment]
Oh okay, got that! thanks! 😁

**Comment by waltermvp** [comment]
@boygirl can you elaborate on the second part im trying this and getting invariant violation errors:

```
          <VictoryLine
            groupComponent={"<G/>"}
            style={{
              data: { stroke: "red" },
              parent: { border: "1px solid #ccc" }
            }}
            data={data1}
            x="weekday"
            y="drinks"
          />
```

**Comment by waltermvp** [comment]
still not working when using 
`"react-native-svg": "^6.1.4",` 

@boygirl

**Comment by boygirl** [comment]
@waltermvp you will need to omit the `^` in your version, as this is breaking with `react-native-svg@6.2.x`

**Comment by boygirl** [comment]
@waltermvp you can also try `groupComponent={<G/>}`. It needs to be an element, not a string.

**Comment by waltermvp** [comment]
@boygirl got it working! thank you very much!

**Comment by boygirl** [comment]
This issue is fixed with `react-native-svg@6.5.0`
